#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-fltk-no-gl.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## DP: do not link against libfltk_gl

@DPATCH@
--- ACE_wrappers.orig/m4/ace.m4
+++ ACE_wrappers/m4/ace.m4
@@ -1180,8 +1180,8 @@
    fi
  fi
  if test X"${FLTKCONFIG}" != X; then
-   ACE_FLTK_CPPFLAGS=`$FLTKCONFIG --use-gl --cxxflags 2>/dev/null`
-   ACE_FLTK_LIBS=`$FLTKCONFIG --use-gl --ldflags 2>/dev/null`
+   ACE_FLTK_CPPFLAGS=`$FLTKCONFIG --cxxflags 2>/dev/null`
+   ACE_FLTK_LIBS=`$FLTKCONFIG --ldflags 2>/dev/null`
 
    AC_SUBST(ACE_FLTK_CPPFLAGS)
    AC_SUBST(ACE_FLTK_LIBS)
